app/src/sharedTest/java/org/oppia/android/app/profile/PinPasswordActivityTest.kt

```diff
@@ -689,6 +691,36 @@ class PinPasswordActivityTest {
     }
   }
 
+  @Test
+  fun testPinPasswordActivity_checkKeyboardType_clickShowHidePassword_keyboardTypeIsSame() {
+    ActivityScenario.launch<PinPasswordActivity>(
+      PinPasswordActivity.createPinPasswordActivityIntent(
+        context,
+        adminPin,
+        adminId
+      )
+    ).use {
+      testCoroutineDispatchers.runCurrent()
+
+      val inputType = InputType.TYPE_CLASS_NUMBER
+      onView(withId(R.id.input_pin)).check(
+        matches(
+          withInputType(
+            inputType
+          )
+        )
+      )
+      onView(withId(R.id.show_pin)).perform(click())
+      onView(withId(R.id.input_pin)).check(
+        matches(
+          withInputType(
+            inputType
+          )
+        )
+      )
+    }
+  }
+
```

Comment:
Also, could you please remove `waitUntilActivityVisible`. It will be helpful as the function has no use.

Reply:
Done